containers/daap-delete-data-product/tests/unit/conftest.py

```diff
@@ -167,16 +167,28 @@ def create_metadata(
 
 
 @pytest.fixture
-def create_glue_database(glue_client, data_product_name):
-    glue_client.create_database(DatabaseInput={"Name": data_product_name})
+def database_names(data_product_name, data_product_versions):
+    return sorted(
+        {
+            data_product_name + "_" + version.split(".")[0]
+            for version in data_product_versions
+        }
+    )
 
 
 @pytest.fixture
-def create_glue_tables(create_glue_database, glue_client, data_product_name):
-    for i in range(3):
-        glue_client.create_table(
-            DatabaseName=data_product_name, TableInput={"Name": f"schema{i}"}
-        )
+def create_glue_databases(glue_client, data_product_name, database_names):
```

Comment:
I think this could be replaced with calls to code in the base image https://github.com/ministryofjustice/data-platform/blob/main/containers/daap-python-base/src/var/task/glue_and_athena_utils.py#L12

Either way, it looks like the data_product_name arg is not needed here.

Reply:
Argument removed and now using the create database code from athena and glue utils